akka_microservices/ibex35Microservice/src/main/scala/com/tapiporla/microservices/retrievers/indices/ibex35/model/Ibex35Stat.scala

```diff
@@ -0,0 +1,47 @@
+package com.tapiporla.microservices.retrievers.indices.ibex35.model
+
+import com.github.nscala_time.time.Imports._
+import com.sksamuel.elastic4s.Hit
+import com.tapiporla.microservices.retrievers.common.model.ElasticDocumentInsertable
+import com.tapiporla.microservices.retrievers.common.stats.StatsGenerator.Stat
+import com.tapiporla.microservices.retrievers.indices.ibex35.dao.Ibex35ESDAO
+
+object Ibex35Stat {
+
+  def fromMap(map: Map[String,String]) = {
+    Ibex35Stat(
+      DateTime.parse(map(Ibex35ESDAO.date), DateTimeFormat.forPattern("dd-MM-yyyy")),
+      map(Ibex35ESDAO.Stats.statsAttr),
+      BigDecimal(map(Ibex35ESDAO.Stats.statsValue))
+    )
+  }
+
+  def fromHit(t: Hit): Ibex35Stat = {
+    val map = t.sourceAsMap
+    Ibex35Stat(
+      DateTime.parse(map(Ibex35ESDAO.date).toString),
+      map(Ibex35ESDAO.Stats.statsAttr).toString,
+      BigDecimal(map(Ibex35ESDAO.Stats.statsValue).toString)
+    )
+  }
+
+  def fromStat(stat: Stat): Ibex35Stat =
+    Ibex35Stat(stat._1, stat._2, stat._3)
```

Comment:
I would use the unapply for tuples in this case

Reply:
Done. Stat is now a case class :)